projects/scion/workbench/src/lib/view-part/view-tab/view-tab.component.ts

```diff
@@ -153,4 +155,30 @@ export class ViewTabComponent {
   public get viewRef(): string {
     return this.view.viewRef;
   }
+
+  /**
+   * Listens for 'dblclick' events to maximize or minimize the application main content, unless closing views quickly.
+   */
+  private installMaximizeListener(zone: NgZone): void {
+    // Install listeners outside the Angular zone to not change detect the application upon every mouse event.
+    zone.runOutsideAngular(() => {
+      let enabled = false;
+
+      merge(fromEvent(this._host, 'mouseenter'), fromEvent(this._host, 'mousemove'), fromEvent(this._host, 'mouseleave'))
+        .pipe(takeUntil(this._destroy$))
+        .subscribe((event: Event) => {
+          enabled = (event.type === 'mousemove');
+        });
+
+      fromEvent(this._host, 'dblclick')
+        .pipe(
+          filter(() => enabled),
+          takeUntil(this._destroy$)
+        )
+        .subscribe((event: Event) => {
+          zone.run(() => this._workbenchLayout.toggleMaximized());
+          event.stopPropagation(); // prevent toggling in view tab bar
```

Comment:
Please change the comment to something like this: 
// prevent \`ViewPartBarComponent\` handling the dblclick event which would undo maximization/minimization

Reply:
done